docs/building/README.md

```diff
@@ -5,20 +5,24 @@ We strongly believe that staying close to browser standards will be the best lon
+2. Bundling and code splitting (combining many into few files)
+3. Transforming/polyfilling standard code for older browsers
+4. Transforming/polyfilling experimental standards (stage 3 proposals) for older browsers
+5. Transforming/polyfilling non-standard code
 
 Our recommended setups address the first four points but refrain from introducing non-standard features. We think it's best to avoid those, as they might lock your source code to a specific build setup which can be hard to get away from at a later time. Javascript is evolving fast, your code can quickly become out of sync with new developments.
 
+## Rollup
+We recommend [Rollup](https://rollupjs.org/) for building front-end projects. Rollup is convenient to use and gets out of your way. It is easy to understand what's going on. Quite a relief in a world of complex javascript tooling.
+
+Rollup is focused on statically analyzing es module based projects, significantly reducing the size of your app through tree shaking, minification and code splitting. Rollup can output es modules, making use of the native module loader available in modern browsers.
+
+Our [default configuration](/building/building-rollup.html) gets your started with a setup for modern browsers, and optionally for supporting legacy browsers.
+
 ## Webpack
-[Webpack](https://webpack.js.org/) is a popular and flexible build tool, with a growing ecosystem of plugins. We recommend it as a general purpose tool. See our [default configuration](/building/building-webpack.html) to get started.
+[Webpack](https://webpack.js.org/) is a popular and flexible build tool, with a large ecosystem of plugins. Out of the box, webpack handles different module formats, such as esm, commonjs, amd and umd. It is not able to output native es modules, relying instead on it's own module loader.
 
-## Rollup
-[Rollup](https://rollupjs.org/) is a another popular build tool which recently had it's 1.0 release. It's especially suitable for optimizing and minifying es modules, and it can output es modules for browsers which have native support. See our [default configuration](/building/building-rollup.html) to get started.
+Compared to rollup it is a more complex tool, with more options for composing and managing complex projects. It also comes with a dev server with features such as auto-reload and hot module reloading.
 
-## Polyfills loader
-Web components are not supported yet on Edge and Internet Explorer 11. The recommended approach is to conditionally load the polyfills at runtime using our [Polyfills Loader](/building/polyfills-loader.html) for more.
+Because webpack is not able to output native es modules, and because it has a steeper learning curve, we don't recommend for general use. However if you have more complex requirements, or because you have to integrate with an existing setup, it can still be a great choice. We have a [default configuration](/building/building-webpack.html) available to get you started.
```

Comment:
```suggestion
Because webpack is not able to output native es modules, and because it has a steeper learning curve, we don't recommend it for general use. However if you have more complex requirements, or because you have to integrate with an existing setup, it can still be a great choice. We have a [default configuration](/building/building-webpack.html) available to get you started.
```